src/RequestBuilder/SortingRequestBuilder.php

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\RequestBuilder;
+
+use Fig\Http\Message\RequestMethodInterface;
+use Lmc\Matej\Model\Command\Interaction;
+use Lmc\Matej\Model\Command\Sorting;
+use Lmc\Matej\Model\Command\UserMerge;
+use Lmc\Matej\Model\Request;
+
+class SortingRequestBuilder extends AbstractRequestBuilder
+{
+    protected const ENDPOINT_PATH = '/sorting';
+
+    /** @var Interaction */
+    private $interactionCommand = null;
+    /** @var UserMerge */
+    private $userMergeCommand = null;
+    /** @var Sorting */
+    private $sortingCommand;
+
+    public function __construct(string $userId, array $itemIds)
+    {
+        $this->sortingCommand = Sorting::create($userId, $itemIds);
```

Comment:
As mentioned, I'd keep this out of the builder and just accept Sorting instances.

Also in the Recommendation builder which will follow, this would require copying all the 5 params, which are required for the Command constructor there...